code/modules/materials/definitions/solids/materials_solid_metal.dm

```diff
@@ -27,6 +27,9 @@
 	lore_text = "A silvery-white metallic chemical element in the actinide series, weakly radioactive. Commonly used as fuel in fission reactors."
 	mechanics_text = "Uranium can be used as fuel in fission reactors."
 	taste_description = "the inside of a reactor"
+	melting_point = 1407
+	boiling_point = 4074
+	ignition_point = null
```

Comment:
Can set this on `/decl/material/solid/` above instead of setting to null on every subtype